auraed/src/cells/cell_service/executables/executable.rs

```diff
@@ -65,17 +64,27 @@ impl Executable {
 
     /// Starts the underlying process.
     /// Does nothing if [Executable] has previously been started.
-    pub fn start(&mut self) -> io::Result<()> {
+    pub fn start(
+        &mut self,
+        uid: Option<u32>,
+        gid: Option<u32>,
+    ) -> io::Result<()> {
         let ExecutableState::Init { command } = &mut self.state else {
             return Ok(());
         };
 
-        let mut child = command
+        let mut command = command
             .kill_on_drop(true)
```

Comment:
Is this variable dropped after this function exits? I'm curious if this setting would cause the cell process to be killed almost immediately, and whether it should be removed.

Reply:
the functionality should be analogous to what happens today.

my understanding is that because we store the child in the `ExecutableState` it isn't dropped.  is that understanding incorrect?

Reply:
Ah yes I missed where we saved the child to our state. 👍🏼